scripts/skills/effects/rf_cheap_trick_effect.nut

```diff
@@ -0,0 +1,122 @@
+this.rf_cheap_trick_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		MeleeSkillModifier = 0,		// Must be set during initialization
+		RangedSkillModifier = 0,
+		DamageMult = 0.0,
+	},
+	function create()
+	{
+		this.m.ID = "effects.rf_cheap_trick";
+		this.m.Name = "Cheap Trick";
+		this.m.Description = "Unleash a quick and deceptive move to catch opponents off-guard, allowing for a higher chance of landing a hit but at the cost of power.";
+		this.m.Icon = "ui/skills/rf_cheap_trick.png";	// TODO
```

Comment:
You'll also need to add `this.m.Overlay` for this so the effect's icon spawns as well.

Reply:
Do we really need to show the same icon, once squared (after using the skill) and then circled (after receiving the effect) on the same character? 
That seems redundant to me

Reply:
Yes because the skill usage shows that the character successfully used the skill and the effect shows that actor got the effect. This is consistent with all similar vanilla skills e.g. Adrenaline, Shieldwall etc. It also future proofs it in the case this effect can be achieved via some other means.